### ISSUE TITLE
Issues in tests with mariaDB & mysql 8 upgrade

### ISSUE THREAD

**Issue by sbrossie** [killbill/killbill#1362]
The mariaDB & mysql 8 upgrade are causing issues when running the test.

This seems to only be happening when running test from IntelliJ (but this makes it hard to follow logs, an find real problems).

```
2020-09-15T21:29:07.249+0000 [notifications-lifecycle-th-1] ERROR jdbc.audit - null. DataSource.getConnection()
java.sql.SQLInvalidAuthorizationSpecException: Could not connect to address=(host=localhost)(port=52960)(type=master) : (conn=135) Access denied for user 'user9a223a54'@'localhost' (using password: YES)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.createException(ExceptionFactory.java:66)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.create(ExceptionFactory.java:153)
	at org.mariadb.jdbc.MariaDbDataSource.getConnection(MariaDbDataSource.java:277)
	at net.sf.log4jdbc.sql.jdbcapi.DataSourceSpy.getConnection(DataSourceSpy.java:147)
	at org.killbill.billing.DBTestingHelper$RetryableDataSource.getConnection(DBTestingHelper.java:130)
	at org.skife.jdbi.v2.DataSourceConnectionFactory.openConnection(DataSourceConnectionFactory.java:40)
	at org.skife.jdbi.v2.DBI$4.execute(DBI.java:230)
	at org.skife.jdbi.v2.DBI$4.execute(DBI.java:227)
	at org.killbill.commons.profiling.Profiling.executeWithProfiling(Profiling.java:35)
	at org.skife.jdbi.v2.DBI.open(DBI.java:227)
	at org.skife.jdbi.v2.DBI.open(DBI.java:219)
	at org.skife.jdbi.v2.DBI.withHandle(DBI.java:300)
	at org.skife.jdbi.v2.DBI.inTransaction(DBI.java:328)
	at org.killbill.queue.DBBackedQueue.executeTransaction(DBBackedQueue.java:351)
	at org.killbill.queue.DBBackedQueueWithPolling.getReadyEntries(DBBackedQueueWithPolling.java:76)
	at org.killbill.notificationq.NotificationQueueDispatcher.getReadyNotifications(NotificationQueueDispatcher.java:261)
	at org.killbill.notificationq.NotificationQueueDispatcher.doDispatchEvents(NotificationQueueDispatcher.java:191)
	at org.killbill.queue.DefaultQueueLifecycle$DispatcherRunnable.dispatchEvents(DefaultQueueLifecycle.java:282)
	at org.killbill.queue.DefaultQueueLifecycle$DispatcherRunnable.access$1000(DefaultQueueLifecycle.java:241)
	at org.killbill.queue.DefaultQueueLifecycle$DispatcherRunnable$1.callback(DefaultQueueLifecycle.java:262)
	at org.killbill.queue.DefaultQueueLifecycle.withHandlingRuntimeException(DefaultQueueLifecycle.java:314)
	at org.killbill.queue.DefaultQueueLifecycle.access$800(DefaultQueueLifecycle.java:42)
	at org.killbill.queue.DefaultQueueLifecycle$DispatcherRunnable.run(DefaultQueueLifecycle.java:258)
	at org.killbill.commons.concurrent.WrappedRunnable.run(WrappedRunnable.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by sbrossie** [comment]
The following workaround was added https://github.com/killbill/killbill/pull/1356/commits/380b1c7d4c3a4ebc7fba7404663234819bae9403

**Comment by pierre** [comment]
Relevant: https://jira.mariadb.org/browse/CONJ-807

**Comment by pierre** [comment]
It looks like https://github.com/killbill/killbill-oss-parent/pull/64 fixed it:

```
➜  grep -i denied logs_before.txt | wc -l
     778
➜  grep -i denied logs_after.txt | wc -l
       0
```
